config/stacks/bsRunsUploadManager.ts

```diff
@@ -0,0 +1,34 @@
+import { BsRunsUploadManagerConfig } from '../../lib/workload/stateless/stacks/bs-runs-upload-manager/deploy/lib/stacks/bs_runs_upload_manager_stack';
+import {
+  devGdsBsRunsUploadLogPath,
+  stgGdsBsRunsUploadLogPath,
+  prodGdsBsRunsUploadLogPath,
+  AccountName,
+} from '../constants';
+
+export const getBsRunsUploadManagerStackProps = (n: AccountName): BsRunsUploadManagerConfig => {
+  const baseConfig = {
+    ica_token_secret_id: 'IcaSecretsPortal',
+    portal_token_secret_id: 'orcabus/token-service-jwt',
+    basespace_token_secret_id: '/manual/BaseSpaceAccessTokenSecret',
+    eventbus_name: '/umccr/orcabus/stateful/eventbridge',
```

Comment:
And, similarly here

```
eventbus_name: eventBusName,
```

Reply:
Resolved in 587c49e24ddffb3091c3b3b158fba9ebcb007f25